core/templates/dev/head/profile/Profile.js

```diff
@@ -19,15 +19,21 @@
  */
 
 oppia.controller('Profile', [
-  '$scope', '$http', '$rootScope',
-  function($scope, $http, $rootScope) {
+  '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
+  function($scope, $http, $rootScope, $oppiaDatetimeFormatter) {
```

Comment:
Even after I took the $ out, I am still having trouble getting a datetime to show. 

Looking at a profile page with a first_contribution_datetime, this code:

<span ng-if="firstContributionDatetime">Contributing since 
                <[getLocaleAbbreviatedDatetimeString(firstContributionDatetime)]></span>

Shows up like this:

Contributing since <[getLocaleAbbreviatedDatetimeString(firstContributionDatetime)]>

Why is that?